package.json

```diff
@@ -6,18 +6,25 @@
   "main": "index.html",
   "scripts": {
     "lint": "eslint . --ext .js,.html",
-    "test": "npm run lint"
+    "test": "npm run lint && npm run test:local",
+    "test:local": "polymer test --skip-plugin sauce",
+    "test:sauce": "polymer test --skip-plugin local"
   },
   "author": "D2L Corporation",
   "license": "Apache-2.0",
   "devDependencies": {
+    "@polymer/test-fixture": "latest",
+    "axe-core": "latest",
     "babel-eslint": "latest",
+    "chai": "latest",
     "demo-transfigurator": "dbatiste/demo-transfigurator",
     "eslint": "latest",
     "eslint-config-brightspace": "latest",
     "eslint-plugin-html": "latest",
     "eslint-plugin-lit": "latest",
-    "polymer-cli": "latest"
+    "mocha": "^5.0.0",
```

Comment:
I needed to lock the version of mocha back a version, otherwise I was seeing console errors.  Can look into this more later